backend/compact-connect/lambdas/python/common/cc_common/data_model/schema/compact/common.py

```diff
@@ -49,3 +62,17 @@ class PaymentProcessorPublicFieldsSchema(Schema):
 
     publicClientKey = String(required=True, allow_none=False)
     apiLoginId = String(required=True, allow_none=False)
+
+
+def validate_no_duplicates_in_configured_states(data):  # noqa: ARG001 unused-argument
+    """Common method to validate that configuredStates list contains no duplicate postal abbreviations."""
+    configured_states = data.get('configuredStates', [])
+
+    configured_state_postal_abbrs = [state['postalAbbreviation'] for state in configured_states]
+    if len(set(configured_state_postal_abbrs)) != len(configured_state_postal_abbrs):
+        duplicate_states = sorted(configured_state_postal_abbrs)
```

Comment:
nit: this a better name?
```suggestion
        sorted_states = sorted(configured_state_postal_abbrs)
```